docs/schema.md

```diff
@@ -387,6 +387,17 @@ The defined database prefixes and their "home" databases are:
         </ul>
       </td>
     </tr>
+    <tr>
+      <td><code>CGA</code></td>
+      <td><a href="https://github.com/wolfi-dev/advisories">CGA / Wolfi - Chainguard Security Notices</a></td>
+      <td>
+        <ul>
+          <li>How to contribute: TBD</li>
```

Comment:
I’d link to a CONTRIBUTING.md in the advisories repo, and then we can flesh that out